src/UserRepository/PdoDatabase.php

```diff
@@ -64,20 +71,32 @@ public function authenticate(string $credential, string $password = null) : ?Use
     /**
      * {@inheritDoc}
      */
-    public function getRolesFromUser(string $username) : array
+    public function getRolesFromUser(string $identity) : array
     {
         if (! isset($this->config['sql_get_roles'])) {
             return [];
         }
 
-        if (false === strpos($this->config['sql_get_roles'], ':username')) {
+        if (false === strpos($this->config['sql_get_roles'], ':identity')) {
             throw new Exception\InvalidConfigException(
-                'The sql_get_roles configuration setting must include a :username parameter'
+                'The sql_get_roles configuration setting must include a :identity parameter'
             );
         }
 
-        $stmt = $this->pdo->prepare($this->config['sql_get_roles']);
-        $stmt->bindParam(':username', $username);
+        try {
+            $stmt = $this->pdo->prepare($this->config['sql_get_roles']);
+        } catch (PDOException $e) {
+            throw new Exception\RuntimeException(sprintf(
+                "Error on sql_get_rols configuration: %s",
```

Comment:
Two things on this one:

- Use single quotes (for consistency)
- I'd rephrase this to 'Error preparing retrieval of user roles: %s'; using "prepare" here helps the developer identify what sort of action was being attempted.